### PR TITLE
Bump golang to v1.13.5

### PR DESCRIPTION
**Pull request by chenrui333**
relates to https://github.com/Homebrew/homebrew-core/pull/47519 (formula is already shipped with go v1.13)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
-
-go_import_path: github.com/sorintlab/stolon
+  - 1.13.x
 
 env:
   - TARGET=amd64
   - TARGET=arm64
 
-install: true
-
```
Comment (chenrui333) [inline .travis.yml]: `vendor` folder has been removed, thus removing this.

File: .travis.yml [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
 script:
-  - GO111MODULE=on make test
```
Comment (chenrui333) [inline .travis.yml]: `GO111MODULE=on` is redundant for go v1.13

**Comment by chenrui333** [comment]
utils.go:260: [sentinel edb15486]: 2019-12-23T21:08:28.736Z INFO cmd/sentinel.go:766 trying to find initial master
utils.go:260: [sentinel edb15486]: 2019-12-23T21:08:28.736Z ERROR cmd/sentinel.go:1895 failed to update cluster data {"error": "cannot choose initial master: no keepers registered"}

**Comment by chenrui333** [comment]
utils.go:298: stopping keeper acb3f673
utils.go:260: [keeper acb3f673]: 2019-12-23T21:08:42.550Z ERROR cmd/keeper.go:1481 failed to start postgres {"error": "postgres exited unexpectedly"}
utils.go:260: [keeper acb3f673]: 2019-12-23T21:08:43.294Z ERROR cmd/keeper.go:678 cannot get configured pg parameters {"error": "dial unix /tmp/.s.PGSQL.2053: connect: no such file or directory"}
utils.go:260: [keeper acb3f673]: 2019-12-23T21:08:44.294Z ERROR cmd/keeper.go:678 cannot get configured pg parameters {"error": "dial unix /tmp/.s.PGSQL.2053: connect: no such file or directory"}

**Comment by sgotti** [comment]
@chenrui333 Sorry but I don't understand this PR and why it's needed. We are also going to remove Travis since we'll rely only on agola.

**Comment by chenrui333** [comment]
yeah, the drive was just to use the latest golang version, I think we already shipped with the latest golang build in homebrew side. 

> We are also going to remove Travis since we'll rely only on Agola CI/CD.

As for removing Travis, I guess, if that is the case, I can help remove it in this PR if the golang upgrade part makes sense.

**Comment by sgotti** [comment]
@chenrui333 I think this PR could be closed since we are removing travis in #750 and all the other changes aren't really needed.

**Pull request closed without merging**